perma_web/functional_tests/test_links.py

```diff
@@ -2,34 +2,56 @@
 
 two_minutes = 120 * 1000
 
-
-def test_create_link(logged_in_user) -> None:
-    """It should be possible to successfully create a link from a URL"""
-    url_field = logged_in_user.locator('#rawUrl')
+def create_link(page):
+    url_field = page.locator('#rawUrl')
     url_field.focus()
     url_field.type("https://example.com/")
-    logged_in_user.locator('#addlink').click()
-    logged_in_user.wait_for_url(re.compile('/[A-Za-z0-9]{4}-[A-Za-z0-9]{4}$'), timeout=two_minutes)
-    assert logged_in_user.title() == 'Perma | Example Domain'
-    assert"Example Domain" in logged_in_user \
+    page.locator('#addlink').click()
+    page.wait_for_url(re.compile('/[A-Za-z0-9]{4}-[A-Za-z0-9]{4}$'), timeout=two_minutes)
+    assert page.title() == 'Perma | Example Domain'
+    assert"Example Domain" in page \
```

Comment:
Just fyi, there's a missing space after `assert` (though it pre-dates this PR)

Reply:
Indeed there is, nice catch. Will fix.